packages/eslint-config-sentry-app/index.js

```diff
@@ -1,3 +1,15 @@
+// TODO(@anonrig): Remove prettier, eslint-config-prettier and eslint-plugin-prettier
+// when Biome integration is complete.
+let isPrettierAvailable = false;
+
+{
+  try {
+    require('prettier')
+  } catch {
+    isPrettierAvailable = false;
+  }
+}
```

Comment:
won't prettier always be installed since we're still using it for yaml and whatever else in git hooks

Reply:
You're right.